data/tutorials/language/0it_01_basic_datatypes.md

```diff
@@ -131,7 +131,7 @@ Indexed access to string characters is possible using the following syntax:
 # "buenos dias".[4];;
 - : char : 'o'
 ```
-Operations on `string` values are provided by the [`Stdlib`](/api/Stdlib.html) and the [`String`](/api/String.html) modules.
+String related operations are provided by the [`Stdlib`](/api/Stdlib.html), the [`String`](/api/String.html), and [`Str`](/api/Stdlib.html) modules.
```

Comment:
The `Str` module is *not* part of the standard library. It is a separate library with known design issues. I don't think it is a good idea to cite it here.

Reply:
I was not aware of the issues. What's planned deprecating or fixing?

Our scope is the OCaml _platform_ not only the standard library. Are you considering removing it from the platform?

Reply:
At the scope of the opam ecosystem, `Str` is not the reference library for regular expression either. 
Moreover, I don't think that the `Str` library has ever been part of the OCaml platform?
The future status of `Str` is mostly in limbo, by lack of people working on it.

Reply:
> At the scope of the opam ecosystem, Str is not the reference library for regular expression either.

We probably don't want to pick an opam package outside of the platform. Except as an example of use. Opam is part of the platform.

> Moreover, I don't think that the Str library has ever been part of the OCaml platform?

Since it is bundled with the compiler and mentioned in [The OCaml API](https://v2.ocaml.org/releases/5.1/api/index.html), I assumed it was.

> The future status of Str is mostly in limbo, by lack of people working on it.

:-(

Reply:
Part of the problem is that writing a modern regexp library (with unicode property classes support) is a very large effort, and such effort is probably better hosted outside of the compiler.
And it is unclear if it would be worthwhile to invest efforts into very slowly updating the Str API to remove its reliance on global state since the end result would be a more usable but still ascii-only (and not PCRE) regexp library.